app/components/Menu/MenuDropdownItem.tsx

```diff
@@ -0,0 +1,131 @@
+import { Pressable, Text, View, type ViewStyle, type TextStyle } from "react-native"
+import { useState, useRef, memo, useCallback } from "react"
+import { themed } from "../../theme/theme"
+import { menuSettings } from "./menuSettings"
+import type { MenuItem } from "./types"
+
+interface MenuDropdownItemProps {
+  item: MenuItem
+  index: number
+  onItemPress: (item: MenuItem) => void
+  onItemHover: (itemLabel: string, index: number, hasSubmenu: boolean) => void
+}
+
+const MenuDropdownItemComponent = ({
+  item,
+  index,
+  onItemPress,
+  onItemHover,
+}: MenuDropdownItemProps) => {
+  const [hoveredItem, setHoveredItem] = useState<string | null>(null)
+  const hoverTimeoutRef = useRef<NodeJS.Timeout | null>(null)
+  const enabled = item.enabled !== false
```

Comment:
Does it make more sense to go with `disabled` rather than `enabled`, so you don't have to check for `false` explicitly, and it aligns better with HTML's `disabled` prop?